spec/index.bs

```diff
@@ -934,9 +934,6 @@ A [=device=] has the following internal slots:
         No [=limit/better=] limits can be used, even if the underlying [=adapter=] can support them.
 </dl>
 
-Issue: [=device=] is already an internal concept. Having slots be marked as internal is redundant
-and adds extra `[[`/`]]` in spec text. Consider removing the brackets.
```

Comment:
Unrelated but I tried this recently and it's not possible unless we switch the internal objects' internal slots to `<dfn dfn>`, because attributes without `[[]]` are treated as real IDL attributes. I guess there is special handling for `[[]]`.